ntbs-service/DataAccess/AlertRepository.cs

```diff
@@ -45,6 +46,15 @@ public async Task<Alert> GetAlertByNotificationIdAndTypeAsync(int? notificationI
                 .SingleOrDefaultAsync(m => m.NotificationId == notificationId && m.AlertType == alertType);
         }
 
+        public async Task<Alert> GetOpenAlertByNotificationIdAndTypeAsync(int? notificationId, AlertType alertType)
+        {
+            return await _context.Alert
+                .Where(m => m.NotificationId == notificationId)
+                .Where(m => m.AlertType == alertType)
+                .Where(m => m.AlertStatus == AlertStatus.Open)
+                .SingleOrDefaultAsync();

```

Comment:
In Jan's original comment I believe he was suggesting just using white space instead of chaining e.g.

```
.SingleOrDefaultAsync(m =>
    m.NotificationId == notificationId
    && m.AlertType == alertType); 
```

Reply:
I was, but I actually quite like this! The final query should be the same anyways, and it's nice and clean.
I'm not sure on `m` as the variable name though... :P